TASVideos/Pages/Publications/Urls/Edit.cshtml.cs

```diff
@@ -49,6 +52,17 @@ public async Task<IActionResult> OnGet()
 		}
 
 		Title = title;
+		var youtubetitle = await db.Publications
+			.Where(p => p.Id == PublicationId)
+			.Select(p => p.YouTubeTitle)
+			.SingleOrDefaultAsync();
```

Comment:
I feel it should be possible to construct yt title on the fly without adding new DB column. By grabbing the correct pub here and just picking its fields that we need to construct the title. This is the only place where it's needed I think, so it doesn't have to exist in the Publication class for everyone (unlike normal title).